articles/resources/relnotes.md

```diff
@@ -17,6 +17,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: August 26th, 2020*
+
+This release contains the following:
+
+- Fixed bug in M.Q.Arithmetic.RippleCarryAdderCDKM.
```

Comment:
We will have a generic line for bug fixes in the release notes, do we want to mention this bug fix in particular? @msoeken @KittyYeungQ